### ISSUE TITLE
Update documentation to setup fake_spectra without cloudy tables

### ISSUE THREAD

**Issue by andreufont** [sbird/fake_spectra#15]
If we follow the instructions in the current setup, we get an error about cloudy tables when importing fake_spectra.gridded_spectra. 

```
Python 3.6.6 |Anaconda, Inc.| (default, Jun 28 2018, 17:14:51) 
[GCC 7.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import fake_spectra
>>> from fake_spectra import griddedspectra
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/font/.conda/envs/desi/lib/python3.6/site-packages/fake_spectra/griddedspectra.py", line 7, in <module>
    from . import spectra
  File "/home/font/.conda/envs/desi/lib/python3.6/site-packages/fake_spectra/spectra.py", line 37, in <module>
    from .cloudy_tables import convert_cloudy
ModuleNotFoundError: No module named 'fake_spectra.cloudy_tables'
```

**Comment by sbird** [comment]
Yes, it got out of date: should be fixed by b5922ab05834

**Comment by andreufont** [comment]
Indeed. Thanks Simeon!
